### PR TITLE
std_misc/process/pipe: Update comments to match current API.

### PR DESCRIPTION
**Pull request by sourcefrog**

### CODE REVIEW

File: examples/std_misc/process/pipe/pipe.rs [hunk 1]
```diff
@@ -16,7 +16,9 @@ fn main() {
     };
 
     {
-        // Write a string to the stdin of `wc`
+        // Write a string to the stdin of `wc`. 
+        // stdin has type `Option<ChildStdin>`, but since we know this instance
```
Comment (steveklabnik) [inline examples/std_misc/process/pipe/pipe.rs]: Two small nits: `stdin` should be in backticks, and there should be an extra `//` between your first and second line.

**Comment by steveklabnik** [comment]
This looks great, just a few small nits

**Comment by sourcefrog** [comment]
Thanks, please take another look.

**Comment by steveklabnik** [comment]
Thank you! :)

**Comment by steveklabnik** [comment]
Hehe whoops, forgot we don't use Bors here! Thanks again.

**Pull request merged**